app/database/accessor/breeze_buddy/call_execution_config.py

```
@@ -286,3 +299,33 @@ async def calling_activation_for_merchant(
+    try:
+        query_text, values = get_all_merchants_query()
+        result = await run_parameterized_query(query_text, values)
+
+        if result:
+            # Extract shop_identifier from each row
+            merchants = [row["shop_identifier"] for row in result]
+            logger.info(f"Found {len(merchants)} unique merchants")
+            return merchants
+
+        logger.info("No merchants found")
+        return []
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)